internal/local/command_kafka_broker_update.go

```diff
@@ -0,0 +1,44 @@
+package local
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/v3/pkg/broker"
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/errors"
+	"github.com/confluentinc/cli/v3/pkg/examples"
+)
+
+func (c *command) newKafkaBrokerUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update <id>",
+		Short: "Update local Kafka broker configurations.",
+		Long:  "Update per-broker configurations.",
```

Comment:
`Long` is shorter than `Short` :thinking:

Reply:
`"Update per-broker configurations for local Kafka brokers."`